### ISSUE TITLE
Webpack 5 does not include polyfills by default

### ISSUE THREAD

**Issue by ctsiaousis** [ctsiaousis/country-chalkboard#2]
This results in undefined references in core modules and can not use packages like "fs" and "path"

**Comment by ctsiaousis** [comment]
seems to be resolved in 14e3a5b32b4ee76726f87c865f59e519178e011f

**Comment by ctsiaousis** [comment]
Still some fail
```
BREAKING CHANGE: The request 'process/browser' failed to resolve only because it was resolved as fully specified
(probably because the origin is strict EcmaScript Module, e. g. a module with javascript mimetype, a '*.mjs' file, or a '*.js' file where the package.json contains '"type": "module"').
The extension in the request is mandatory for it to be fully specified.
Add the extension to the request.
```

**Comment by ctsiaousis** [comment]
This was about mixing up server and client side code,
since this project is a static website the file parsing is done with AJAX

marking as closed after 52d098723bd63e68e99c35db388f0e57110bb9e2
